### ISSUE TITLE
Problems during running simulation

### ISSUE THREAD

**Issue by IndigoFloyd** [PEESEgroup/PFAL-DRL#1]
Your work is remarkable! I was really inspired by this paper. However, when I ran the whole year simulation, I encountered some problems as follows. The Python environment complies with your README documentation.
### 1. In `drl_test.py`, `baseline_test.py` and `main_simulation.py`
I found that when the simulator initializes the state using `x = env.reset()`, `x` will be initialed as a tuple like `([states], {an empty info dict})`. As a result, `u = policy(Batch(obs=[x], info=info)).act[0].cpu().detach().numpy()` throws an error because `x` should have homogeneous elements.

Therefore, I initialized `x` in this way: `x = env.reset()[0]`, which resolved this error.

### 2. In `PFALEnv.py`
I noticed that although the simulation could run, it never stopped even when I kept it running for over 2 days. It seemed to be stuck in January for a prolonged time. Then I looked into the flag variable `done`. The logic for the termination condition
`done_term = not (
      # xx[0] < self.DESIRED_LETTUCE_WEIGHT
      self.k < self.t_max or status == 0)`looked unusual.

Therefore, I changed the logic to set `done_term` to `True` when either condition was satisfied:
`done_term = (self.k >= self.t_max) or (status != 0)`

This fix allows me to generate the result matrices.

## Weird Results
After running a whole year simulation for Dubai, I tried to compute the total $$CO_2$$ usage. I added an item `c_CO2`, and divided it by its price `self.c_CO2 = 0.2` to compute the summary usage during a step. But I didn't get the same result as the Fig. 3(a).

I'm not sure if I've done something incorrectly. Thank you for your help in resolving these issues!

**Comment by decardinb** [comment]
@IndigoFloyd Thanks for the interest in our work. I just saw these comments hence the delay in response. I have fixed the environment reset. The fix for your done_term and the original one should produce the same results. I have however, updated to your fix to make it clearer and easier to read. Thanks for the suggestions.

The values in Fig.3(a) are in kg/kg FW and not just the total carbon dioxide. I hope this helps.

**Comment by IndigoFloyd** [comment]
> [@IndigoFloyd](https://github.com/IndigoFloyd) Thanks for the interest in our work. I just saw these comments hence the delay in response. I have fixed the environment reset. The fix for your done_term and the original one should produce the same results. I have however, updated to your fix to make it clearer and easier to read. Thanks for the suggestions.
> 
> The values in Fig.3(a) are in kg/kg FW and not just the total carbon dioxide. I hope this helps.

Thank you for your fixes and reply! I think it will work well.
